low-level/files/opt/seagate/sspl/setup/sspl_test.py

```diff
@@ -0,0 +1,62 @@
+#!/bin/env python3
+
+# Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published
+# by the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <https://www.gnu.org/licenses/>.
+# For any questions about this software or licensing,
+# please email opensource@seagate.com or cortx-questions@seagate.com.
+
+import os
+import sys
+import errno
+import argparse
+import inspect
+import traceback
+import subprocess
+
+# Add the top level directories
+sys.path.insert(0, '/opt/seagate/cortx/sspl/low-level')
+from cortx.sspl.lowlevel.framework.base.sspl_constants import PRODUCT_FAMILY
+from cortx.sspl.lowlevel.files.opt.seagate.sspl.setup.sspl_setup import Cmd as SSPLSetup
```

Comment:
Not supposed to import sspl_setup in here. Any common functions in sspl_setup will be moved to cortx-utils or sspl utility.py, and needs to be used from there. Refer post-install PR
https://github.com/Seagate/cortx-monitor/pull/279/files

Reply:
Done.